README.md

```diff
@@ -23,3 +73,15 @@ docker run -p 3000:3000 tacoma/fqm-execution-service:latest # Run server on port
 ### Local
 
 Run the service with `npm start`. By default, it will run on port 3000. To change the port, do `PORT=XXXX npm start`
+
+## License
+
+Copyright 2021 The MITRE Corporation
```

Comment:
This work started in 2020, so this should be changed. Also I just checked the license file and the (c) statement is blank so please fill that in too

Reply:
I'll change the copyright date here. Technically the one in the LICENSE file is supposed to be blank; it's a template for how to use the Apache 2.0 (aka "copy this into your project and change this date").

Reply:
Technically correct, the best kind of correct.